features/view-site-in-lang-i18n-l10n.feature

```diff
@@ -0,0 +1,35 @@
+Feature: As a visitor
+  In order to view the site in my language
+  I need to be able choose either Swedish or English for the site
+
+  PT: https://www.pivotaltracker.com/story/show/133316647
+
+  Background:
+    Given I am Logged out
+
+
+  Scenario: Default language is Swedish
+    Given I am on the "all companies" page
+    Then I should see "Hitta H-märkt företag"
+    And I should not see "Swedish flag"
+    And I should see "English flag"
+    And I should see t("theme_copyright", locale: :sv)
+
+
+  Scenario: Visitor switches the site language from English to Swedish
+    Given I am on the "all companies" page
+    When I click on "change-lang-to-english"
+    When I click on "change-lang-to-svenska"
```

Comment:
I don't know why, it's @weedySeaDragon's steps. But I'm sure there is some kind of logic behind it. =)

And now I figured out what it was, the css is there and in the steps so that if changed to images instead of text links the steps will still pass. Probably.